azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+
+jobs:
+- job: Test
+  strategy:
+    matrix:
+      node_12: 
+        node_version: '12'
+        TEST_REPOSITORY: 'CORE'
+      node_10:
+        node_version: '10'
+        TEST_REPOSITORY: 'CORE'
+      node_8:
+        node_version: '8'
+        TEST_REPOSITORY: 'CORE'
+  variables:
+    TEST_REPOSITORY: $(test_repository)
+  pool:
+    vmImage: 'ubuntu-latest'
```

Comment:
Can we used a fixed version so the CI would be reproducible ?

Reply:
Yes, of course, I will add 12.2.0, 10.15.3 and 8.16.0